src/Microsoft.Framework.Configuration.Abstractions/IConfigurationRoot.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.Framework.Configuration
+{
+    public interface IConfigurationRoot : IConfiguration
```

Comment:
So its not that its that complicated, but it will cause someone to wonder what the differences are between the 3, and there doesn't appear to be much which is why I thought it was a bit weird...